CONFIG_REFERENCE.md

```diff
@@ -194,25 +197,56 @@ specify them explicitly!)
   - `src_hostname`
     Hostname reported by the client in the EHLO/HELO command.
 
-  See below for example.
+    Example:
+    ```
+    match rcpt_domain emersion.fr {
+        filter dkim verify
+        deliver local
```

Comment:
So this will deliver locally e-mails that are for both local and non-local recipients?

Reply:
And it is most likely to fail because the `local` module has no idea what to do with non-local recipients. Can you help me pick a better usage example here? (or should I remove it altogether?)

Reply:
I was about to suggest a silly example like "if the date matches YYY-12-24, add a X-Merry-Christmas header field" but we don't even have the required directives to do so. Probably better not to have an example for now.

Reply:
Oh, can you remove this example?

Reply:
Removed.